src/abstractdataframe/io.jl

```diff
@@ -165,6 +165,59 @@ end
 
 ##############################################################################
 #
+# LaTeX output
+#
+##############################################################################
+
+function latex_char_escape(char::SubString)
```

Comment:
I still think this should be `Char` or `AbstractString`. Either is fine, though the function and argument name suggest to me that `Char` would make the most sense.